client/src/components/JoinMeeting.test.tsx

```diff
@@ -4,18 +4,18 @@
 import { PrimaryButton, TextField } from '@fluentui/react';
 import { mount } from 'enzyme';
 import { generateTheme } from '../utils/ThemeGenerator';
-import { JoinTeamsMeeting } from './JoinTeamsMeeting';
+import { JoinMeeting } from './JoinMeeting';
```

Comment:
we can do a for each for all the test cases in this file and add for rooms meeting link as well

Reply:
Let's discuss! I don't think I completely understand what you're suggesting 😃

Reply:
Done!